### PR TITLE
update provisioning tool

### PR DESCRIPTION
**Pull request by stevenBorisko**
update provisioning tool with tfvars

### CODE REVIEW

File: scripts/train/run_batch.sh [hunk 1]
```diff
@@ -13,32 +13,38 @@
...
-  echo NAME_PREFIX $NAME_PREFIX
-  echo ZONE $ZONE
-  echo INSTANCE_COUNT $INSTANCE_COUNT
-  echo GPU_COUNT $GPU_COUNT
-  echo VM_TYPE $VM_TYPE
-  echo ACCELERATOR_TYPE $ACCELERATOR_TYPE
-  echo IMAGE_FAMILY_NAME $IMAGE_FAMILY_NAME
-  echo IMAGE_NAME $IMAGE_NAME
-  echo DISK_SIZE_GB $DISK_SIZE_GB
-  echo DISK_TYPE $DISK_TYPE
```
Comment (soumyapani) [inline scripts/train/run_batch.sh:28]: lets add the missing variables as well.

**Comment by stevenBorisko** [comment]
@soumyapani pointed out to me that I omitted `DISK_SIZE_GB, DISK_TYPE, GCS_MOUNT_LIST, NFS_FILESHARE_LIST`. I don't see those set anywhere in the repo -- do y'all need those conversions added?

**Review by soumyapani — CHANGES_REQUESTED** [review]

File: scripts/train/run_batch.sh [hunk 2]
```diff
@@ -13,32 +13,38 @@
...
-  echo ACTION $ACTION
-  /usr/entrypoint.sh
+  echo '```tfvars'
+  cat "${TFVARS_FILE}"
+  echo '```'
+  /root/aiinfra/scripts/entrypoint.sh \
+      ${MINIMIZE_TERRAFORM_LOGGING:+ --quiet} \
+      ${TERRAFORM_GCS_PATH:+ --backend-bucket ${TERRAFORM_GCS_PATH}} \
+      ${ACTION,,} \
+      mig \
```
Comment (Chris113113) [inline scripts/train/run_batch.sh:46]: Is this supposed to be hardcoded to `mig`?

**Comment by Chris113113** [comment]
GKE side of things LGTM, will defer to @abdallag for the training side changes.

**Review by Chris113113 — APPROVED** [review]

File: scripts/train/run_batch.sh (hunk 2 above)
Comment (stevenBorisko) [reply scripts/train/run_batch.sh:46]: The only clusters available are `["gke", "mig", "mig-with-container", "slurm"]`; the other script seemed like it was specifically for `gke`, and variables required for `mig-with-container` and `slurm` are not present in this script, so I kinda just assumed it to be specifically for `mig`. I might be missing something though, should I make it variable?

**Comment by abdallag** [comment]
Later version of the tool was integrated in PR#68

**Pull request closed without merging**